Code/Engine/GameEngine/GameState/Implementation/GameState.cpp

```diff
@@ -125,9 +125,22 @@ void ezGameState::CreateActors()
       ezView* pView = nullptr;
       EZ_VERIFY(ezRenderWorld::TryGetView(m_hMainView, pView), "");
       ezUniquePtr<ezActor> pXRActor = pXRInterface->CreateActor(pView, ezGALMSAASampleCount::Default, std::move(pMainWindow), std::move(pOutput));
-      ezActorManager::GetSingleton()->AddActor(std::move(pXRActor));
```

Comment:
I guess the commented out code 20 lines above could go now. Also this piece could be moved into a helper function.

Reply:
Code was removed already after rebase.